packages/ui-library/src/components/input-field-number/index.stories.ts

```diff
@@ -607,7 +607,7 @@ export const HasError = (params: BlrInputFieldNumberType) => {
 /**
  * ## Dependencies
  * ### Button Icon
- * The Input Field Number component makes use of the Button Icon component for increasing or decreasing the value. For more information have a look at the [Button Icon](?path=/docs/design-system-web-components-actions-buttons-button-icon--docs) component.
+ * The Input Field Number component makes use of the Button Icon component for increasing or decreasing the value. For more information have a look at the [Button Icon](?path=/docs/components-actions-buttons-button-icon--docs) component.
```

Comment:
The Button Icon link is not working

Reply:
done

Reply:
Sorry @bar-tay , the whole Button Icon section was removed with #1015 so this review was not up to date anymore. I see that you added the section back to the documentation, this is however not needed anymore. Can you remove it again?

Reply:
done